src/main/java/com/amazon/opendistroforelasticsearch/security/filter/OpenDistroSecurityRestFilter.java

```diff
@@ -155,4 +238,11 @@ private boolean checkAndAuthenticateRequest(RestRequest request, RestChannel cha
         
         return false;
     }
+
+    @Subscribe
+    public void onWhitelistingSettingChanged(WhitelistingSettingsModel whitelistingSettingsModel) {
+        WhitelistingSettings whitelistingSettings = whitelistingSettingsModel.getWhitelistingSettings();
```

Comment:
Instead of managing default values here could default value checks be moved to WhitelistingSettingsImpl?

Reply:
Yes, I have made this change.